x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/conversations/conversations.spec.ts

```diff
@@ -68,69 +68,72 @@ export default function ApiTest({ getService }: FtrProviderContext) {
                 conversation: conversationUpdate,
               },
             },
-          })
-          .expect(404);
+          });
+        } catch (error) {
+          expect(error.status).to.be(404);
+        }
       });
 
       it('returns a 404 for retrieving a conversation', async () => {
-        await observabilityAIAssistantAPIClient
-          .slsEditor({
+        try {
+          await observabilityAIAssistantAPIClient.editor({
             endpoint: 'GET /internal/observability_ai_assistant/conversation/{conversationId}',
             params: {
               path: {
                 conversationId: 'my-conversation-id',
               },
             },
-          })
-          .expect(404);
+          });
+        } catch (error) {
+          expect(error.status).to.be(404);
+        }
       });
     });
 
-    describe('when creating a conversation with the write user', function () {
+    describe('when creating a conversation with the write user', () => {
       let createResponse: Awaited<
         SupertestReturnType<'POST /internal/observability_ai_assistant/conversation'>
       >;
-
       before(async () => {
-        createResponse = await observabilityAIAssistantAPIClient
-          .slsEditor({
-            endpoint: 'POST /internal/observability_ai_assistant/conversation',
-            params: {
-              body: {
-                conversation: conversationCreate,
-              },
+        createResponse = await observabilityAIAssistantAPIClient.editor({
+          endpoint: 'POST /internal/observability_ai_assistant/conversation',
+          params: {
+            body: {
+              conversation: conversationCreate,
             },
-          })
-          .expect(200);
+          },
+        });
+        expect(createResponse.status).to.be(200);
       });
 
       after(async () => {
-        await observabilityAIAssistantAPIClient
-          .slsEditor({
-            endpoint: 'DELETE /internal/observability_ai_assistant/conversation/{conversationId}',
-            params: {
-              path: {
-                conversationId: createResponse.body.conversation.id,
-              },
+        const { status } = await observabilityAIAssistantAPIClient.editor({
+          endpoint: 'DELETE /internal/observability_ai_assistant/conversation/{conversationId}',
+          params: {
+            path: {
+              conversationId: createResponse.body.conversation.id,
             },
-          })
-          .expect(200);
+          },
+        });
+        expect(status).to.be(200);
 
-        await observabilityAIAssistantAPIClient
-          .slsEditor({
+        try {
+          await observabilityAIAssistantAPIClient.editor({
```

Comment:
Feels good getting rid of the serverless-specific client. Is the plan to get rid of `x-pack/test_serverless/api_integration/test_suites/observability/ai_assistant/common/observability_ai_assistant_api_client.ts` entirely?

Reply:
We can remove it entirely if we get all the tests running in deployment agnostic, unless there is some case where we want a serverless only test because of some serverless only feature.  Probably wouldn't hurt to leave it.